tests/unit/v2/test_transformer.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import
+
```

Comment:
can you move this file to https://github.com/aws/sagemaker-python-sdk/tree/zwei/tests/unit/sagemaker/cli/compatibility/v2?